packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -157,6 +167,12 @@ const DialoguesList = ({ classes }: { classes: ClassesType }) => {
     <p>Beta feature: Dialogues between a small group of users. Click to see more.</p>
   </div>
 
+  const renderMyDialogues = myDialogues?.length 
+
+  const myDialoguesTooltip = <div>
+      <div>These are dialogues that you are a coauthor on (including drafts and published dialogues).</div>
```

Comment:
Do we want this to have the same language as the (same?) tooltip on the /dialogues page?